test/test_torch.py

```diff
@@ -17955,92 +17955,155 @@ def test_case_info(fn_name, config):
                         f'Subprocess exception while attempting to run {test_case_info(fn_name, config)}:\n'
                         + e.output.decode("utf-8")))
 
-    @onlyCPU
-    @dtypes(*(torch.testing.get_all_complex_dtypes() + [torch.float, torch.double]))
-    def test_addbmm(self, device, dtype):
-        # num_batches = 10
-        # M, N, O = 12, 8, 5
-        num_batches = 2
-        M, N, O = 2, 3, 4
-        b1 = torch.randn(num_batches, M, N, dtype=dtype, device=device)
-        b2 = torch.randn(num_batches, N, O, dtype=dtype, device=device)
-        res = torch.bmm(b1, b2)
-        res2 = torch.tensor((), dtype=dtype, device=device).resize_as_(res[0]).zero_()
-        res3 = torch.tensor((), dtype=dtype, device=device).resize_as_(res[0]).zero_()
+    def _test_addbmm_baddbmm(self, func, b1, b2, ref, res2):
```

Comment:
"out_tensor" seems like a clearer name than "res2" since there's no "res1" and "res" is an unclear abbreviation

Reply:
fixed